larpandoracontent/LArThreeDReco/LArCosmicRay/CosmicRayTrackRecoveryAlgorithm.cc

```diff
@@ -1,722 +0,0 @@
-/**
```

Comment:
I'm not sure why this file has disappeared with this commit. Was this intended?

Reply:
Yes, I added some visualisations to fo some debugging and I think I must have added the file at some point with whitespace changes

Reply:
I had added some code for debugging some failure and accidentally added it to the PR, i think when trying to remove it i accidentally deleted it. But now adding it back in!